### DISCUSSION TITLE
How to make scheduling on adonis?

### DISCUSSION THREAD

**Discussion by mkevison** [adonisjs/core#3901] [Help] [answered]
Hello guys!

I need to schedule requests in an API and manipulation in the database.  I'm not able to integrate the Cron node library in adonis

**Comment by Julien-R44** [comment, marked as answer]
Hello!

You can find a very complete article on this subject here : 
https://tech.ndianabasi.com/comprehensive-guide-for-creating-and-managing-cron-jobs-in-nodejs-adonisjs-applications

Head here if your decision is to use a library. Here it's node-schedule but it's exactly the same with node-cron :
https://tech.ndianabasi.com/comprehensive-guide-for-creating-and-managing-cron-jobs-in-nodejs-adonisjs-applications#heading-step-6-install-node-schedule-package

**Reply by mkevison** [reply]
Thank you so much, that's what I needed!
